builder/azure/arm/config.go

```diff
@@ -371,8 +371,8 @@ type Config struct {
 	AllowedInboundIpAddresses []string `mapstructure:"allowed_inbound_ip_addresses"`
 
 	// Runtime Values
-	UserName               string
-	Password               string
+	UserName               string `mapstructure:",exclude"`
```

Comment:
I'm wondering if `skip` is better than `exclude` 🤔

Reply:
I think exclude is fine. "skip" already has meaning, so I think it would be confusing.

Reply:
may be this could be: `hcl2-exclude` ?

Reply:
I think this need to be in another tag, for less confusion: in this branch that will be abandoned: https://github.com/hashicorp/packer/compare/hcl2_singular_blocks?expand=1#diff-c758e6371295fa09c7bcf506ea29fb9a I used the `mapstructure-to-hcl2` tag; this tag could have a `,skip` option 🙂 what do you think ?

Reply:
I saying this because this fields won't be excluded or skipped for mapstructure.

Reply:
I was trying to find this cause you told me about it in person, but I didn't know it's in another branch! I will check this out, thank you!